community/user-stories/index.md

```diff
@@ -0,0 +1,105 @@
+
+### Matthew Alsup - Software Engineer - Asurion
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/_BPKWGQa0y0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+### Alex King - Director of Tech Ops - Enova
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/t158qOctb1c" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+###  Nick Whittier - Senior Engineering Manager - HelloSign
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/n07FXiFNTO0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+###  Mark Gustafson - Open Connect Operations Team - Netflix
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/LxwKTXUq3Hg" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+### Tony Drane - Engineering Team - Pure Storage
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/v82Mahv3Ri8" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+### Anne Cesa Klein - SRE - Pure Storage
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/sCDbS5bAoWU" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+###  Maleko Taylor - DevOps Engineer - LaunchDarkly
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ccOeMn1X9O0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+###  Bernard Golden - Vice President of Cloud Strategy - Capital One
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Bn_FaJMf_4I" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+###  Maarten Dirkse - Cloud Advocate - Bol.com
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/gIV4tAtC_7E" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+###  Lilit Yenokyan - Director of Engineering - Pivotus
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/y9RRmTon-UY" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+##  User Stories from people building & supporting Spinnaker:
+
+### Andy Glover - Dir. Delivery Engineering - Netflix
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/OFuCsax7BIk" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+### Emily Burns - Sr. Software Engineer - Netflix
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/PwT8xYjq870" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+###  Robert Zienert - Sr. Software Engineer - Netflix
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/LdRicdyLFp8" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+### Thomas Lin - Sr. Software Engineer - Netflix
```

Comment:
s/Thomas/Tomás/

Reply:
Would it be possible to also change the spelling of his name on video?

Reply:
@dorbin yes I'll get that fixed